### DISCUSSION TITLE
点击新的对话没有响应

### DISCUSSION THREAD

**Discussion by xiaohe331** [GaiZhenbiao/ChuanhuChatGPT#860] [Q&A]
我部署在本地以后，在打开的页面中点击新的对话，对话并不会全部清除。不知道这个和我部署的是本地模型，删除了使用openai key相关的代码有没有关系？

**Comment by xiaohe331** [comment]
我屏蔽js代码以后，可以清空了，但是不知道会不会有其他影响？ 
![image](https://github.com/GaiZhenbiao/ChuanhuChatGPT/assets/129476028/acb4f996-3d66-4a89-a1d5-b5cfbf33c4ce)

**Comment by Keldos-Li** [comment]
希望您能提供具体的错误表现，“对话并不会全部清除”，那么哪些被清除了，哪些没被清除，还是没有相应？理论上不会有这个问题，js是为了清楚只读的对话历史……不应该注释掉

在我的测试中，本地模型的对话记录能够正常使用这个按钮清空。也许这个问题和gradio版本和python版本有关。

如果去掉js又可以了，maybe说明js的输入输出阻塞了Python代码的运行，应该会在终端有报错提示。

另外，可以把该段改为
```python
    emptyBtn.click(
        reset,
        inputs=[current_model],
        outputs=[chatbot, status_display],
        show_progress=True,
        _js = 'clearHistoryHtml'
    )
```
试一试。

**Reply by xiaohe331** [reply]
谢谢，我现在从仓库拉取了最新的代码，已经没有这个问题了
